pytorch_lightning/trainer/training_loop.py

```diff
@@ -269,6 +269,15 @@ def _check_training_step_output(self, training_step_output):
             if training_step_output.grad_fn is None:
                 # TODO: Find why - RuntimeError: Expected to mark a variable ready only once ...
                 raise MisconfigurationException("In manual optimization, `training_step` should not return a Tensor")
+        elif self.trainer.automatic_optimization:
```

Comment:
```suggestion
        elif self.trainer.lightning_module.automatic_optimization:
```
tests are failing because of this attribute access